transcribe.py

```diff
@@ -44,10 +44,14 @@ def main(args):
 	data_paths = [path for path in data_paths if os.path.basename(path) not in exclude]
 
 	labels, model, decoder = setup(args)
-	val_dataset = datasets.AudioTextDataset(data_paths, [labels], args.sample_rate, frontend = None, segmented = True, mono = args.mono, time_padding_multiple = args.batch_time_padding_multiple, audio_backend = args.audio_backend, speakers = args.speakers) 
+	val_dataset = datasets.AudioTextDataset(data_paths, [labels], args.sample_rate, frontend = None, segmented = True, mono = args.mono, time_padding_multiple = args.batch_time_padding_multiple, audio_backend = args.audio_backend, speakers = args.speakers, exclude=exclude)
+	examples = len(val_dataset.examples)
```

Comment:
Please rename to `num_examples`. Have you tried using `len(val_dataset)` instead of accessing the private field?

Reply:
Renamed. I have not tried

Reply:
Have you checked what `AudioTextDataset.__len__` does? Pleas use `len(val_dataset)`, this is more idiomatic and avoids accessing private fields for no reason